### DISCUSSION TITLE
Initial release

### DISCUSSION THREAD

**Discussion by af** [af/hyperpaper-planner#2] [Announcements]
Just threw up the current pdf I had after people asked on Reddit, but I'm still tweaking things and trying to improve the design. Let me know what works & what doesn't, suggestions for improvement, etc either here or in a separate Discussion thread

<hr /><em>This discussion was created from the release <a href='https://github.com/af/planner-pdf/releases/tag/'>Initial release</a>.</em>

**Comment by dmhowcroft** [comment]
This is an extremely cool project. Do you have a donate button where I can throw money at you if I find it useful after a couple of months?

**Reply by af** [reply]
Thank you! No donate button but my tentative plan is to sell a customizable full 2022 version at a reasonable cost. Would be happy to accept your donation then :D Planning on incorporating feedback into subsequent versions of the Nov/Dec 2021 version over the course of this month

**Comment by sndierlamm** [comment]
Wow, that ist really cool! I am looking forward to the further development!
